components/core-initializers/src/main/java/org/eclipse/dirigible/components/initializers/synchronizer/SynchronizationInitializer.java

```diff
@@ -53,6 +64,7 @@ public void handleContextStart(final ApplicationReadyEvent are) {
 		synchronizationProcessor.prepareSynchronizers();
 		classpathExpander.expandContent();
 		synchronizationProcessor.processSynchronizers();
+		initializers.forEach(Initializer::initialize);
```

Comment:
maybe initialize() could throw runtime exception for one of the Initializers?

Reply:
you’re right, I’ll update this